tests/v1/test_waku_connect.nim

```diff
@@ -58,9 +58,9 @@ procSuite "Waku connections":
       p2 = await n2.rlpxConnect(newNode(n3.toENode()))
       p3 = await n4.rlpxConnect(newNode(n3.toENode()))
     check:
-      p1.isNil
-      p2.isNil == false
-      p3.isNil == false
+      p1.isErr()
```

Comment:
In other cases it is xplicitly checking against `true` - should we be consistent and do it here as well?